### DISCUSSION TITLE
关于添加 Anthropic Claude 模型的讨论与投票

### DISCUSSION THREAD

**Discussion by BennyThink** [BurnHair/docs#204] [调查问卷]
很多人都希望头顶冒火能够支持 Claude 系列模型，但是一直以来没来得及添加，主要有以下原因：

# 1. 没有价格合适、且靠谱的渠道
我自己是有 Anthropic 的账号的，但是并没有任何优惠，只能按照官方的价格使用。

更进一步，**由于税务要求，价格会比官方汇率高出10-20%左右**，按照目前 $1 = ¥7.12 的汇率来看，可能最终会是 **$1 = ¥8** 左右

<img width="1547" alt="iShot_2024-10-26_12 54 50" src="https://github.com/user-attachments/assets/59525f97-eb27-4ec5-b467-c5cb9b6d3172">

我知道有很多价格低廉的渠道，但是不是很有信心。

# 2. one-api 被我改坏了
可能已经不太支持 Claude了，不过这个应该好说，直接再跑一个 newapi 就好了

# 3. 滥用
滥用的事情很麻烦。Anthropic 的封号非常非常非常严格。

如果再因为有人滥用，导致我的 Anthropic 账号被封禁，那么后续处理会非常痛苦。我又没有很多时间可以处理这些事情。

--------------

有鉴于此，在这里希望大家能够投票并留下自己的想法。我们会根据投票结果来决定未来的路线，包括但不限于 Anthropic Claude、Gemini 等

**Comment by Godfatherofpython** [comment]
或许可以参考一下Deepbricks的呀，不确定这个建议如何呢

**Comment by notdp** [comment]
我有一个账号开的claude pro用了半年，api冲了40刀还没怎么用，pro就直接封号了，pro的封号基本是实时发起退款，隔天就到了，但是api还没封，封号半个月了，到现在还能用，这就很迷。

**Reply by kangjian99** [reply]
pro和API两套逻辑应该是，我很早就在用Claude API，但一直没敢充 pro。

**Comment by zhang19980518** [comment]
希望佬增加

**Comment by zakizou** [comment]
服务非常的稳定快速，希望盈利，未来可期

**Comment by fevrax** [comment]
支持

**Comment by YersiniaHerb** [comment]
从one-api升级成chatnio是不是也行

**Comment by BennyThink** [comment]
上线咯

https://github.com/orgs/BurnHair/discussions/207
